sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2344,18 +2344,36 @@ object GpuOverrides extends Logging {
         override def convertToGpu(childExprs: Seq[Expression]): GpuExpression =
           GpuSum(childExprs.head, a.dataType)
       }),
+    expr[NthValue](
+      "nth aggregate operator",
+      ExprChecks.fullAgg(
```

Comment:
This says that we support nth for reductions and group by aggregations too. If that is true we have some thing things that we need to update to deal with this. If not we should reflect just what we support.